examples/ml_pipeline.md

```diff
@@ -0,0 +1,208 @@
+# Intro
+This document will show an example of running the ML pipeline all the way through,
+from data prep to training to inference. This document is focused on running scripts
+rather than interfacing with the API itself.
+
+# Preparing Experimental Data
+The first step in the pipeline is to download and parse the experimental data from CDD.
+This will act as our labels when training, and will also give us the SMILES strings to
+dock when generating our structural inputs.
+## Download and Parse Data
+First we will download and parse the CDD data. Both of these steps are handled in the
+same script:
+`asapdiscovery-data/asapdiscovery/data/scripts/download_moonshot_data.py`. Running this
+script will require a CDD API token, which can either be stored in the `CDDTOKEN`
+environment variable, or in a file that is then passed to the script. By default, the
+script will download all Moonshot compounds that have fluorescence values for SARS-Cov-2
+Mpro, but other searches can be specified using the CLI args.
+
+As an example, we will download the default search, keeping all molecules that are
+achiral or enantiopure and including semiquantitative data. We assume in this case that
+the `CDDTOKEN` env variable has been set appropriately. This call also saves the
+unfiltered and unprocessed file as a cache file to avoid repeated CDD download calls:
+```bash
+python /path/to/data/scripts/download_moonshot_data.py \
```

Comment:
Are these not package scripts that you can just run like `download_moonshot_data`? Longterm these could become part of the mega cli

Reply:
yep, good point. I updated all the script calls to just use the CLI package call. would definitely be good to add everything to `click` at some point, but that's something for later I think